package.json

```diff
@@ -21,13 +21,24 @@
   ],
   "license": "MIT",
   "scripts": {
+    "precommit": "lint-staged",
+    "postcommit": "git update-index --again",
     "build": "gatsby build",
     "develop": "gatsby develop",
     "start": "yarn run develop",
     "format": "prettier --write \"src/**/*.js\"",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "devDependencies": {
+    "babel-eslint": "9.x",
+    "eslint": "5.x",
+    "eslint-config-react-app": "^3.0.5",
+    "eslint-plugin-flowtype": "2.x",
```

Comment:
Do we need flowtype? I'd be happy not to use flow with this project. I found it hindered more than helped in MOMO.

Reply:
It is a required peerDependency of eslint-config-react-app